//#ifndef MAINWINDOW_H
//#define MAINWINDOW_H

//#include <cameraclient.h>
//#include <QMainWindow>
//#include <QMessageBox>
//#include <QScopedPointer>
//#include <QHostAddress>

//namespace Ui {
//class MainWindow;
//}

//class MainWindow : public QMainWindow
//{
//    Q_OBJECT

//public:
//    explicit MainWindow(const QString ipPort, QWidget *parent = 0);
//    ~MainWindow();

//private slots:
//    void on_sendMessagePushButton_clicked();

//    void on_controllerParamsPushButton_clicked();

//private:
//    Ui::MainWindow *ui;
//    QScopedPointer <Camera> camHard;
//    QScopedPointer <CameraClient> camera;

//};

//#endif // MAINWINDOW_H
